src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -76,13 +76,13 @@ public class LaboratoryController {
 	// TODO: to centralize
 	protected static final String DEFAULT_PAGE_SIZE = "80";
 	
-	private static String draft = LaboratoryStatus.DRAFT.toString();
+	private static String draft = LaboratoryStatus.draft.toString();
 	
-	private static String open = LaboratoryStatus.OPEN.toString();
+	private static String open = LaboratoryStatus.open.toString();
 
-	private static String deleted = LaboratoryStatus.DELETED.toString();
+	private static String deleted = LaboratoryStatus.deleted.toString();
 	
-	private static String invalid = LaboratoryStatus.INVALID.toString();
+	private static String invalid = LaboratoryStatus.invalid.toString();
```

Comment:
```suggestion
	private static String invalid = LaboratoryStatus.invalid.toString();
	
	private static String done = LaboratoryStatus.done.toString();
```

Reply:
As `done` is a `static` shouldn't it be:
<pre>
private static final String DONE = LaboratoryStatus.done.toString();
</pre>

Reply:
yes, you are right, all of them

Reply:
@ArnaudFonzam I've applied myself the suggested changes, please pull and align the rest. Thanks!